spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/saml2/Saml2RelyingPartyProperties.java

```diff
@@ -300,11 +317,11 @@ public void setBinding(Saml2MessageBinding binding) {
 				this.binding = binding;
 			}
 
-			public boolean isSignRequest() {
+			public Boolean isSignRequest() {
```

Comment:
I thinktThis should be deprecated with the original signature and a getter style should be added (considering the wrapper type).

Reply:
Actually maybe that without deprecating it since we're going to roll back to that once the support of the old format is removed.